android/src/test/java/com/adyenreactnativesdk/configuration/CardConfigurationParserTest.kt

```diff
@@ -0,0 +1,94 @@
+package com.adyenreactnativesdk.configuration
+
+import com.adyen.checkout.card.AddressConfiguration
+import com.adyen.checkout.card.KCPAuthVisibility
+import com.adyen.checkout.card.SocialSecurityNumberVisibility
+import com.adyen.checkout.card.data.CardType
+import com.facebook.react.bridge.ReadableArray
+import org.junit.Assert.*
+import org.junit.Test
+import org.mockito.Mockito.mock
+import org.mockito.Mockito.`when`
+
+class CardConfigurationParserTest {
+
+    @Test
+    fun testGetConfiguration() {
+    }
+
+    @Test
+    fun testGetBcmcConfiguration() {
+    }
+
+    @Test
+    fun testGetShowStorePaymentField() {
+        val config = WritableMapMock()
+        config.putBoolean(CardConfigurationParser.SHOW_STORE_PAYMENT_FIELD_KEY, false)
+        val cardParser = CardConfigurationParser(config, "US")
+        assertFalse(cardParser.showStorePaymentField)
+    }
+
+    @Test
+    fun testGetHolderNameRequired() {
+        val config = WritableMapMock()
+        config.putBoolean(CardConfigurationParser.HOLDER_NAME_REQUIRED_KEY, true)
+        val cardParser = CardConfigurationParser(config, "US")
+        assert(cardParser.holderNameRequired)
```

Comment:
Should that be assertTrue or shall we keep assert?

Reply:
Fixed on [6398b8](https://github.com/Adyen/adyen-react-native/pull/304/commits/6398b8a6cd6fb093dab5052969224955f3f88dc7)